client/lxd_instances.go

```diff
@@ -1213,7 +1217,10 @@ func (r *ProtocolLXD) ExecInstance(instanceName string, exec api.InstanceExecPos
 				_, _, _ = conn.ReadMessage() // Consume pings from server.
 			}()
 
-			go args.Control(conn)
+			if args.Control != nil {
```

Comment:
Just wondering whether it would make sense to provide a default handler if it is not set? Otherwise, users need to write it for themselves because we do not expose it (it is part of lxc/exec command).

Reply:
Thanks for clarification 👍